src/test/java/com/adobe/target/edge/client/service/DefaultTargetServiceTest.java

```diff
@@ -137,6 +139,56 @@ public void testExecuteNotification() {
         .execute(any(Map.class), any(String.class), any(DeliveryRequest.class), any(Class.class));
   }
 
+  /**
+   * If a user sends a notifications request with Context.beacon = true, we should always set it to
+   * false for them. Beacon does not make sense to use with a server-side SDK.
+   */
+  @Test
+  public void testExecuteNotificationWithBeaconTrue() {
+
+    ResponseWrapper<DeliveryResponse> mockedResponseWrapper = getTestDeliveryResponse();
+    getMockedTelemetry();
+    Mockito.lenient()
+        .doReturn(mockedResponseWrapper)
+        .when(targetHttpClient)
+        .execute(any(Map.class), any(String.class), any(DeliveryRequest.class), any(Class.class));
+
+    TargetDeliveryRequest targetDeliveryRequestMock = getDeliveryRequest();
+    targetDeliveryRequestMock.getDeliveryRequest().getContext().setBeacon(Boolean.valueOf(true));
```

Comment:
No need to cast to Boolean, Java autoboxing/unboxing will take care of that.
```suggestion
    targetDeliveryRequestMock.getDeliveryRequest().getContext().setBeacon(true);
```